### PR TITLE
Fix Bedrock notebook & add streaming example

### PR DESCRIPTION
**Pull request by leilacc**
Now that Bedrock is GA'd I was able to actually run this notebook. Found a few little bugs and added a streaming example.

### CODE REVIEW

File: notebooks/bedrock/Query command XL.ipynb [hunk 1]
```diff
@@ -100,13 +100,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install --upgrade cohere-aws\n",
+    "!pip install --upgrade cohere-aws boto3\n",
```
Comment (jpekmez) [inline notebooks/bedrock/Query command XL.ipynb]: why do we need boto3 here? can't we specify boto3 version in dependencies?

File: notebooks/bedrock/Query command XL.ipynb [hunk 2]
```diff
@@ -74,16 +75,15 @@
...
    "cell_type": "code",
-   "execution_count": 18,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
     "import os\n",
     "\n",
     "# Replace with your credentials\n",
     "os.environ['AWS_ACCESS_KEY_ID'] = \"your key id\"\n",
```
Comment (jpekmez) [inline notebooks/bedrock/Query command XL.ipynb]: if I remember, `aws configure` in the [quick start](https://github.com/cohere-ai/cohere-aws#quick-start) is enough?

**Review by jpekmez — DISMISSED** [review]

File: setup.py [hunk 3]
```diff
@@ -24,15 +24,15 @@ def has_ext_modules(foo) -> bool:
 
 
 setuptools.setup(name='cohere-aws',
-                 version='0.8.4',
+                 version='0.8.6',
```
Comment (leilacc) [inline setup.py:27]: I accidentally released 0.8.5 already

File: notebooks/bedrock/Query command XL.ipynb (hunk 1 above)
Comment (leilacc) [reply notebooks/bedrock/Query command XL.ipynb]: ah, yep!

File: notebooks/bedrock/Query command XL.ipynb (hunk 2 above)
Comment (leilacc) [reply notebooks/bedrock/Query command XL.ipynb]: you're right - didn't realize those instructions were there

**Review by jpekmez — APPROVED** [review]

**Pull request merged**